src/components/AddBook.jsx

```diff
@@ -1,10 +1,33 @@
+// import { useState, useEffect } from 'react';
+import AddButton from './AddButton';
+
 function AddBook() {
   return (
     <form className="add-book-form">
       <h2>ADD NEW BOOK</h2>
-      <input type="text" name="book-name" id="book-name" placeholder="Book title" />
-      <input type="text" name="book-author" id="book-author" placeholder="Author" />
-      <button type="submit">Add Book</button>
+      <input
+        type="text"
+        id="book-name"
+        placeholder="Book Title"
+        // onChange={(e) => { setTitle(e.target.value); }}
+      />
+      <br />
+      <input
+        type="text"
+        id="book-author"
+        placeholder="Author"
+        // onChange={(e) => { setAuthor(e.target.value); }}
+      />
+      <br />
+      <input
+        type="text"
+        id="book-category"
+        placeholder="Category"
+        // onChange={(e) => { setCategory(e.target.value); }}
+      />
+      <br />
+      <span className="error">Please enter all three feilds</span>
```

Comment:
- [ ] Kindly consider uncommenting the commented-out lines and refactoring your code so that your form should 
have local state for title and author, which is updated through the onChange event. the usage and understanding of controlled form in react is necessary in this milestone.